lfs.h

```diff
@@ -240,6 +239,14 @@ struct lfs_config {
     // larger attributes size but must be <= LFS_ATTR_MAX. Defaults to
     // LFS_ATTR_MAX when zero.
     lfs_size_t attr_max;
+
+    // Whether read and validate after writing. The positive number means
+    // enable while the negative number means disable. The zero means
+    // enable for compatible. It can imporve write performance if no
+    // validation, but drivers must ensure data integrity.
+    int validate;
+#define LFS_ENABLE_VALIDATE 1
+#define LFS_DISABLE_VALIDATE -1
```

Comment:
What are these defines up to? It looks like they're unused.

Reply:
They are defined for user to assign option validate. To me, using these macro is more clarity than number.